src/main/cc/wfa/virtual_people/common/field_filter/utils/type_convert_util.cc

```diff
@@ -0,0 +1,121 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "wfa/virtual_people/common/field_filter/utils/type_convert_util.h"
+
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/numbers.h"
+#include "absl/strings/str_cat.h"
+#include "google/protobuf/descriptor.h"
+
+namespace wfa_virtual_people {
+
+template <typename ValueType>
+absl::StatusOr<ValueType> ConvertToNumeric(const std::string& input);
+
+template <>
+absl::StatusOr<int32_t> ConvertToNumeric<int32_t>(const std::string& input) {
```

Comment:
absl::string_view please (also in other places in this file)

Reply:
Done.